### ISSUE TITLE
Chinese/Japanese characters

### ISSUE THREAD

**Issue by maniasso** [XoruX/apps-docker#3]
Hello, I noticed other bug in the Docker image, when I go to lpars, in some title it shows some chinese/japanese characters.

**Comment by maniasso** [comment]
![image (1)](https://user-images.githubusercontent.com/43616714/94145060-5a0b1e80-fe48-11ea-8155-16f0b1460d00.png)

**Comment by goricz** [comment]
Hello, are there some strange characters in that LPAR name? Other titles are OK? Can you hard refresh your browser (Ctrl+F5) and check again?

**Comment by maniasso** [comment]
Yes, it´s in the lpar name. even refreshing the browser this happen, I tried other browsers and same issue.
This doesn´t happen if lpar2rrd is deployed under a Red Hat or Centos, only happen in the docker image.

**Comment by goricz** [comment]
Can you please copy & paste here original LPAR name which is incorrectly displayed? I'll simulate it in our Docker instance, thanks.

**Comment by maniasso** [comment]
hello, one example is m42p-r-ashlpa-7a6c7dd8-000000d2

Em sex., 25 de set. de 2020 às 10:08, goricz <notifications@github.com>
escreveu:
